tensilelite/Tensile/SolutionStructs.py

```diff
@@ -2668,28 +2683,30 @@ def assignDerivedParameters(state):
       state["UnrollMajorLDSMetadata"] = state["TransposeLDSMetadata"] and (not state["ProblemType"]["TLUMetadata"])
 
     auto_LdsBlockSizePerPadA_for_mix = 0
+    tmpBpe = state["ProblemType"]["DataTypeA"].numBytes() if state["ConvertAfterDS"] else state["ProblemType"]["DataType"].numBytes()
     if state["LdsBlockSizePerPadA"] == -1:
       auto_LdsBlockSizePerPadA_for_mix = 1
       if state["UnrollMajorLDSA"]:
-        state["LdsBlockSizePerPadA"] = roundUpToNearestMultiple(state["_DepthUA"] * state["ProblemType"]["DataType"].numBytes(), 128)
-        if state["_DepthUA"] * state["ProblemType"]["DataType"].numBytes() * state["VectorWidthA"] > 128:
-          state["LdsBlockSizePerPadA"] = roundUpToNearestMultiple(state["_DepthUA"] * state["ProblemType"]["DataType"].numBytes() * state["VectorWidthA"], 128)
+        state["LdsBlockSizePerPadA"] = roundUpToNearestMultiple(tmpBpe, 128)
```

Comment:
Should it be **state["_DepthUA"]*tmpBpe** or **tmpBpe**?

Reply:
update